models/CoronSegmentator/scripts/usd_design/usd_create.py

```
@@ -0,0 +1,70 @@
+    def create_usd(self):
+        """Create a USD file from the input mesh files and apply materials."""
+        logger.info(f"Starting <{self.input_files}> USD creation process.")
+        try:
+            # Convert the input STL mesh files to USD format.
+            creator = USDMeshCreator(self.input_files, self.output_file)
+            mesh_paths = creator.convert_stl_to_usd()
+
+            # Apply materials from a shader file to the generated USD meshes.
+            applier = USDShaderApplier(self.output_file, self.shader_file)
+            applier.apply_materials_to_mesh(mesh_paths)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)